### PR TITLE
EWPP-418: Render webtools op publication list media using media container.

### PR DESCRIPTION
**Pull request by 22Alexandra**

### CODE REVIEW

File: tests/Kernel/MediaRenderTest.php [hunk 1]
```diff
@@ -250,4 +258,25 @@ public function testDocumentMedia(): void {
     }
   }
 
+  /**
+   * Tests Webtools Op Publication List media rendering.
```
Comment (sergepavle) [inline tests/Kernel/MediaRenderTest.php]:
```suggestion
   * Tests Webtools OP Publication List media rendering.
```

File: tests/Kernel/MediaRenderTest.php [hunk 2]
```diff
@@ -250,4 +258,25 @@ public function testDocumentMedia(): void {
...
+   */
+  public function testWebtoolsOpPublicationList(): void {
+    // Create a op publication list media.
```
Comment (sergepavle) [inline tests/Kernel/MediaRenderTest.php]:
```suggestion
    // Create a OP Publication list media.
```

**Review by sergepavle — DISMISSED** [review]

File: tests/Kernel/MediaRenderTest.php [hunk 3]
```diff
@@ -250,4 +258,25 @@ public function testDocumentMedia(): void {
...
+      'oe_media_webtools' => '6313',
+    ]);
+    $media->save();
+
+    $build = $this->mediaViewBuilder->view($media, 'default');
+    $html = $this->renderRoot($build);
+    $crawler = new Crawler($html);
+    $figure = $crawler->filter('figure.ecl-media-container');
+    $this->assertCount(1, $figure);
+    $this->assertCount(1, $crawler->filter('div.ecl-media-container__media.ecl-media-container__media--ratio-16-9'));
```
Comment (brummbar) [inline tests/Kernel/MediaRenderTest.php]: Test that the JSON is inside the rendered html.

**Review by brummbar — CHANGES_REQUESTED** [review]

**Review by nagyad — DISMISSED** [review]

**Comment by nagyad** [comment]
Approved, still 417 needs to go in before.

**Review by brummbar — DISMISSED** [review]

**Review by imanoleguskiza — DISMISSED** [review]

**Review by imanoleguskiza — APPROVED** [review]

**Review by brummbar — APPROVED** [review]

**Pull request merged**